src/OpenTelemetry/Metrics/Exemplar/ExemplarReservoir.cs

```diff
@@ -8,33 +8,33 @@ namespace OpenTelemetry.Metrics;
 /// </summary>
 internal abstract class ExemplarReservoir
 {
+    /// <summary>
+    /// Gets a value indicating whether or not the <see
+    /// cref="ExemplarReservoir"/> should reset its state when performing
+    /// collection.
```

Comment:
It would be good to mention that this would be `true` when the AggregationTemporality is Delta and `false` otherwise. It would help with understanding the Collect code better.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/5386/commits/4e2f42982aeaf194e9ba3f8b8ec254b9a1d7aca5

Added but I really don't know if it helps make anything clearer 😄

Reply:
My thinking behind adding that was if you're looking at the Collect code, the IntelliSense for this property would quickly tell us what it means. One could always find that out even if without the comments but this would reduce the number of keystrokes/ mouse clicks needed to get that information. 😄